lib/pages/chat_details/chat_details.dart

```diff
@@ -20,7 +20,8 @@ import 'package:fluffychat/widgets/matrix.dart';
 enum AliasActions { copy, delete, setCanonical }
 
 class ChatDetails extends StatefulWidget {
-  const ChatDetails({Key? key}) : super(key: key);
+  final String roomId;
+  const ChatDetails({Key? key, required this.roomId}) : super(key: key);
```

Comment:
`const ChatDetails({super.key, required this.roomId});`

Reply:
Fixed